modules/tinymce/src/core/test/ts/browser/selection/MultiClickSelectionTest.ts

```diff
@@ -1,32 +1,162 @@
+import { Cursors, Mouse } from '@ephox/agar';
 import { describe, it } from '@ephox/bedrock-client';
+import { SugarElement } from '@ephox/sugar';
 import { TinyAssertions, TinyHooks, TinySelections } from '@ephox/wrap-mcagar';
+import { assert } from 'chai';
 
+import DOMUtils from 'tinymce/core/api/dom/DOMUtils';
 import Editor from 'tinymce/core/api/Editor';
+import { findClosestBlockRange } from 'tinymce/core/selection/MultiClickSelection';
+
+import * as ViewBlock from '../../module/test/ViewBlock';
+
+type RangePath = [startContainerPath: number[], startOffset: number, endContainerPath: number[], endOffset: number];
 
 describe('browser.tinymce.core.selection.MultiClickSelectionTest', () => {
-  const hook = TinyHooks.bddSetupLight<Editor>({
-    base_url: '/project/tinymce/js/tinymce'
-  }, [], true);
+  describe('findClosestBlockRange test', () => {
+    const DOM = DOMUtils.DOM;
+    const viewBlock = ViewBlock.bddSetup();
+    const getRoot = viewBlock.get;
+    const setupHtml = viewBlock.update;
+
+    const resolveRange = (args: RangePath): Range => {
+      const root = SugarElement.fromDom(getRoot());
+      const startContainer = Cursors.calculateOne(root, args[0]);
+      const endContainer = Cursors.calculateOne(root, args[2]);
```

Comment:
Is there a reason we're not using `Cursors.range()` here, as then you wouldn't need the custom `RangePath` type and could just use `Cursors.CursorPath`. That would also make things a lot more readable below as you'd have named parts instead of trying to remember what's the start path, offset, etc...

- [ ] Consider changing this to use `CursorPath` and `Cursors.range`

Reply:
Such a way to specify ranges was suggested [here](https://github.com/tinymce/tinymce/pull/7726#discussion_r838417439) by @spocke . And it turns out this is a matter of taste.

Reply:
What I've said doesn't contradict what Spocke said and instead expands on it further. In general we prefer named structures instead of unnamed as you can't tell what each array item represents easily.